ide/utils/mailinglist.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import mailchimp
+
+from django.conf import settings
+
+
+mailchimp_api = mailchimp.Mailchimp(apikey=settings.MAILCHIMP_API_KEY)
+mailchimp_default_list_id = settings.MAILCHIMP_LIST_ID
+
+
+def add_user(user, mailing_list_id=None):
+    if mailchimp_api is None:
+        raise KeyError("Missing MAILCHIMP_API_KEY environment variable")
```

Comment:
Is this going to cause logging in to crash if `MAILCHIMP_API_KEY` is undefined? That should not be fatal.